operator/cmd/gke_addon_poststart/main.go

```diff
@@ -39,6 +40,20 @@ spec:
   mode: namespaced
 `
 
+const defaultConfigConnectorNamespaceSeparated = `
+apiVersion: core.cnrm.cloud.google.com/v1beta1
+kind: ConfigConnector
+metadata:
+  name: configconnector.core.cnrm.cloud.google.com
+  labels:
+    tenancy.gke.io/access-level: supervisor
+    tenancy.gke.io/project: no-project
+    tenancy.gke.io/tenant: no-tenant
```

Comment:
Can I ask more about these 3 labels? I want to know the life cycle of them, e.g. how is it used/modified, etc.

Reply:
Answered in corporate chat